content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -42,7 +42,7 @@ kubelets.
 
 #### 2. Install the RuntimeClass CRD
 
-The RuntimeClass [CustiomResourceDefinition][] (CRD) can be found in the addons directory of the
+The RuntimeClass [CustomResourceDefinition][/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/] (CRD) can be found in the addons directory of the
```

Comment:
There's already a link reference on line 52. If you want to change the link, can you make the change there or delete it?